platform/net/ubuntu_net_manager.go

```diff
@@ -137,7 +142,20 @@ func (net UbuntuNetManager) SetupNetworking(networks boshsettings.Networks, errC
 
 	staticAddresses, dynamicAddresses := net.ifaceAddresses(staticConfigs, dhcpConfigs)
 
-	err = net.interfaceAddressesValidator.Validate(staticAddresses)
+	// Skip the validation of virtual interfaces(like eth0:0, eth0:1... )
+	staticAddressesWithoutVirtual := []boship.InterfaceAddress{}
+	r, err := regexp.Compile(`:\d+`)
+	if err != nil {
+		return bosherr.WrapError(err, "There is a problem with your regexp.")
```

Comment:
Be specific about which Regexp that is failing, e.g., `"There is a problem with your regexp: `:\d+` that is used to skip validation of virtual interfaces, e.g., eth0:0, eth0:1, ..."` With that you may even be able to remove comment above :)